src/addin/client-interfaces/addin-client-open-help-args.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Interface for launching the help tab from an add-in.
+ */
+export interface AddinClientOpenHelpArgs {
+
+  /**
+   * The name of the help key to be launched.
+   */
+  helpKey?: string;
```

Comment:
any reason not to require helpKey?  I.e., why allow null?

Reply:
My mistake, actually. I was intending to follow the navigate pattern, which is non-nullable, but I think I looked at the openModalArgs IF by mistake, which was nullable. I will fix it and update.